diagnostics-app/src/app/app.component.html

```diff
@@ -5,3 +5,4 @@
 -->
 
 <h1>ChromeOS Diagnostics App</h1>
+<button (click)="toggleTheme()">Switch Theme</button>
```

Comment:
Switch Theme might not be understood by all users. Let's be specific and call it something like "Enable Dark Mode" or just "Dark Mode".

Reply:
Yeah this looks much better. 👍🏻

Reply:
@MahmoudAGawad I've included a demo video in the PR description.

Reply:
I cannot see the video

Reply:
My bad. Forgot to update the comment. It's visibile now! :)

Reply:
I see it now. But it is just changing the text color and not the background color. Is that the expected behavior?

Reply:
Yes there's no background as of now. The entire app is just the header component. The layout will be added in latter PRs. The color of background will be change then.

Reply:
if this is the case, then let's wait until the layout PR is merged and then merge this one. After this PR is merged, change theme should be working as expected.

Reply:
No actually this PR is necessary for the next ones. We can merge this for now 👍 The dark theme setup we do here, and the header component in the next PR, both are necessary for the app layout one.

Reply:
Offline discussion: Decided to add a background color that will be flipped with dark mode button toggle.